src/test/lib/pgSQL-tests.js

```diff
@@ -0,0 +1,425 @@
+                pg.execute(
+                    dropTable, [],
+                    (err,res) => {
+                         xcallback(err, res);
+                    }
+                );
+            },
+            () => { pg.close(); done();}
+        );
+
+    });      
+
+    describe('1.Execute', function () {
+
+        it ('Should give error on invalid query', function (done) {
+            querystr='add new TABLE repositories;';
+
+            pg.execute(querystr, [], function (err,res){
+                assert.notEqual(err,null);
+                assert.ok(res);
+                done();
+            })
+
+
+        });
+
+        it ('Should give no errors with seprate params in statement', function (done) {
+            var param=[999, 'Firstname','domain','911','TOken'];
+            querystr="INSERT INTO repositories (id,name,domain,contact,token) VALUES ($1,$2,$3,$4,$5) RETURNING *";
+            pg.execute(querystr, param, function (err,res){
+                assert.ok(res);
+                assert.equal(err,null);  
+                assert.equal(res.length, 1);
+                assert.equal(res[0].id, 999);
+                assert.equal(res[0].name, 'Firstname');
+                assert.equal(res[0].domain, 'domain');
+                assert.equal(res[0].contact, '911');
+                assert.equal(res[0].token, 'TOken');       
+                done();
+            })
+        });
+
+        it ('Should give error with invalid params', function (done) {
+            var param=['Firstname','domain','911','TOken'];
+            querystr="INSERT INTO repositories (id,name,domain,contact,token) VALUES ($1,$2,$3,$4,$5)";
+            pg.execute(querystr, param, function (err,res){
+                assert.notEqual(err,null);           
+                done();
+            })
+        });
+
+           
+
+    });
+
+
+    describe('2.Insert.',function () {
+
+        it ('Should give no errors with correct statement', function (done) {
+           //INSERT INTO repositories VALUES (99, 'Firstname','domain','911','TOken')
+    
+               pg.insert({id:100, name:'Firstname',domain:'domain', contact:'911', token:'TOken'}, 'repositories', function (err,res){
+                    assert.equal(err,null);
+                    assert.equal(res.length, 1);
+                    assert.equal(res[0].id, 100);
+                    assert.equal(res[0].name, 'Firstname');
+                    assert.equal(res[0].domain, 'domain');
+                    assert.equal(res[0].contact, '911');
+                    assert.equal(res[0].token, 'TOken');                    
+                   // console.log(res);
+                    done();
+                })
+        });
+
+        it ('Should give error on invalid insert', function (done) {
+            //INSERT IN repositories VALS (99, 'Firstname','domain','911','TOken')
+                pg.insert((100,'Firstname','domain','911','TOken'), 'repositories', function (err,res){
```

Comment:
spacing. one level less indentation
```suggestion
            pg.insert((100,'Firstname','domain','911','TOken'), 'repositories', function (err,res) {
```